index.js

```diff
@@ -228,14 +233,25 @@ exports.createServer = function(proxy) {
 
 		if (!route) return onproxy();
 
-		route(domain, function(err, to) {
+		route(routeData, function (err, to) {
+			var toIp,
+				ttl;
```

Comment:
some indentation thing looks off here. in general i prefer multiple var statements but if you fix the indentation thats fine too